src/Core/Grid/Query/LinkBlockQueryBuilder.php

```diff
@@ -109,12 +114,21 @@ private function getQueryBuilder(array $filters)
                 continue;
             }
 
+            if ('id_shop' === $name) {
+                $qb
+                    ->andWhere("lbs.id_shop IN (:$name)")
+                    ->setParameter($name, $value, Connection::PARAM_STR_ARRAY);
+                ;
+
+                continue;
+            }
+
             $qb
                 ->andWhere(sprintf('lbl.%s LIKE :%s', $name, $name))
                 ->setParameter($name, '%' . $value . '%')
             ;
         }
-
+        
```

Comment:
```suggestion
```

I think php-cs-fixer is needed here :D

Reply:
I think we need to use csfixer from PrestaShop de tools on a whole module, but shall I do it now or in separated PR after finishing multi-store?

Reply:
If you plan on running it on all the module, it's better to do it in a different PR, otherwise, this one will get hard to read (imho)